build/azure-pipelines/common/extract-telemetry.sh

```diff
@@ -4,10 +4,25 @@ set -e
 cd $BUILD_STAGINGDIRECTORY
 git clone https://github.com/microsoft/vscode-telemetry-extractor.git
 cd vscode-telemetry-extractor
-git checkout f538e3157c84d1bd0b239dfc5ebccac226006d58
+git checkout 615c5491625341c4276b0e256dd8d6b014d8051b
 npm i
-npm run setup-extension-repos
-node ./out/cli-extract.js --sourceDir $BUILD_SOURCESDIRECTORY --excludedDirPattern extensions  --outputDir . --applyEndpoints --includeIsMeasurement --patchWebsiteEvents
```

Comment:
im assuming this is intended to be gone `--patchWebsiteEvent` and not replaced by something else below but just calling it out

Reply:
Yeah, because that adds in website events which code.visualstudio.com sends and isn't related to the vs code application